medical_pathology/models/medical_pathology.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import _, api, fields, models
+from openerp.exceptions import ValidationError
+
+
+class MedicalPathology(models.Model):
+    _name = 'medical.pathology'
+    _description = 'Medical Pathology'
+
+    name = fields.Char(
+        required=True,
+        translate=True,
+    )
+    code = fields.Char(
+        required=True,
+    )
+    code_type_id = fields.Many2one(
+        string='Code Type',
+        comodel_name='medical.pathology.code.type',
+        index=True,
+    )
+    notes = fields.Text(
+        translate=True,
+    )
+    category_id = fields.Many2one(
```

Comment:
`category_ids` - A pathology should be able to relate to multiple category types. It should, however, verify unique `category.code_type_id` so that only one code type per category is introduced